src/scores/continuous/consistent_impl.py

```diff
@@ -0,0 +1,285 @@
+        S(x,y) = \\frac{1}{2}(\\phi(y)-\\phi(\\kappa_v (x-y) + y) + \\kappa_v (x-y)\\phi'(x))
+
+    where
+        - :math:`\\kappa_v(x) = \\mathrm{max}(-v, \\mathrm{min}(x, v))` is the "capping function"
+        - :math:`v` is the Huber parameter
+        - :math:`x` is the forecast
+        - :math:`y` is the observation
+        - :math:`\\phi` is a convex function of a single variable
+        - :math:`\\phi'` is the subderivative of :math:`\\phi`
+        - :math:`S(x,y)` is the score.
+
+    The score is negatively oriented with :math:`0 \\leq S(x,y) < \\infty`.
+
+    We strongly recommend that you work through the tutorial
+    https://scores.readthedocs.io/en/stable/tutorials/Consistent_Scores.html to help you
+    understand how to use the consistent scoring functions in scores.
+
+    Args:
+        fcst: array of forecast values.
+        obs: array of corresponding observation values.
+        huber_param: Huber mean tuning parameter. This corresponds to the transition point between
+            linear and quadratic loss for Huber loss. Must be positive.
+        phi: a convex function on the real numbers, accepting a single array like argument.
+        phi_prime: a subderivative of `phi`, accepting a single array like argument.
+        reduce_dims: Optionally specify which dimensions to reduce when
+            calculating the consistent Huber score. All other dimensions will be preserved. As a
+            special case, 'all' will allow all dimensions to be reduced. Only one
+            of `reduce_dims` and `preserve_dims` can be supplied. The default behaviour
+            if neither are supplied is to reduce all dims.
+        preserve_dims: Optionally specify which dimensions to preserve when calculating
+            the consistent Huber score. All other dimensions will be reduced. As a special case, 'all'
+            will allow all dimensions to be preserved. In this case, the result will be in
+            the same shape/dimensionality as the forecast, and the errors will be the consistent Huber
+            score at each point (i.e. single-value comparison against observed), and the
+            forecast and observed dimensions must match precisely. Only one of `reduce_dims`
+            and `preserve_dims` can be supplied. The default behaviour if neither are supplied
+            is to reduce all dims.
+        weights: Optionally provide an array for weighted averaging (e.g. by area, by latitude,
+            by population, custom)
+
+    Returns:
+        array of (mean) scores that is consistent for Huber mean functional,
+        with the dimensions specified by `dims`. If `dims` is `None`, the returned DataArray will have
+        only one entry, the overall mean score.
+
+    Raises:
+       ValueError: if `huber_param <= 0`.
+
+    References:
+        -   Taggart, R. J. (2022a). Point forecasting and forecast evaluation with
+            generalized Huber loss. Electronic Journal of Statistics, 16(1).
```

Comment:
This reference (Taggart 2022a) doesn't have page numbers, when the other references in the consistent scores APIs do.

Reply:
Thanks - added

Reply:
Maybe this is my misunderstanding, but it looks like you removed the "a" in the reference (from "(2022a)" to "(2022)"). But earlier in the docstring it says "See Taggart (2022a), or Equation (11) from Taggart (2022b). See Taggart (2022b), end of Section 3.4, for the standard formula." Does this matter?

Reply:
Thanks -added the a back in